src/patternfly/components/Dropdown/dropdown-toggle-button.hbs

```diff
@@ -0,0 +1,11 @@
+<button class="pf-c-dropdown__toggle-button{{#if dropdown-toggle-button--modifier}} {{dropdown-toggle-button--modifier}}{{/if}}"
+  id="{{dropdown-toggle-button--id}}"
+  aria-label="Actions"
```

Comment:
I think the label "Actions" is fine here for the Dropdown component example (and I also would like to see this example evolve to resemble a traditional split button when we add that css, and not show the check), but for the toolbar demo, this label should be a better description of the group of actions that you would get. I think just "Select" would be sufficient in that example.